pi_piper.gemspec

```diff
@@ -24,8 +24,8 @@ Gem::Specification.new do |s|
 
```

Comment:
Alright updated `extra_rdoc_files` to only include non-ruby files. And removed they keys you mentions except I kept rubygems_version. I see some other popular gems have it.